ucsm_apis/server/boot.py

```diff
@@ -101,33 +67,95 @@ def boot_policy_create(handle, name, org_dn="org-root",
                       descr=descr)
 
     mo.set_prop_multiple(**kwargs)
-    handle.add_mo(mo, modify_present=True)
+    handle.add_mo(mo, True)
```

Comment:
Did this earlier to bring it in line with others API. Now others APIs changed so bring this back in sync too with others.